platform-includes/debug-symbols-apple/_default.mdx

```diff
@@ -95,6 +95,11 @@ Another option is to use warnings, and then set `GCC_TREAT_WARNINGS_AS_ERRORS` t
 <OrgAuthTokenNote />
 
 ```bash {tabTitle:Warn on failures - nonblocking}
+#Uncomment the following lines if you installed sentry-cli via Homebrew in a Apple Silicon Mac
```

Comment:
`m`: Does this do any damage to non-Apple Silicon Macs? If not, we could simply run the code for all users. Uncommenting this code isn't the best user experience IMO.

Reply:
> Does this do any damage to non-Apple Silicon Macs?

It shouldn’t. But I haven't tested it.